logstash-scheduler/src/main/java/org/apache/mesos/logstash/config/LogstashConfig.java

```diff
@@ -9,9 +9,16 @@
 @Component
 @ConfigurationProperties(prefix = "logstash")
 public class LogstashConfig {
+
+    private static final String DEFAULT_EXECUTOR_IMAGE_NAME = "mesos/logstash-executor";
```

Comment:
I see what you're thinking but until we have a suge for that feature keeping code at a minimum is our priority. So please inline.